go/vt/logutil/logger.go

```diff
@@ -231,6 +231,13 @@ func (ml *MemoryLogger) String() string {
 	return buf.String()
 }
 
+// Clear clears the logs.
+func (ml *MemoryLogger) Clear() {
```

Comment:
Okay. Works for me. That's why I marked it as optional and LGTM'd the change overal :)